blazor/treegrid/rows/detail-template.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: post
+title: Detail Template in Blazor Tree Grid Component | Syncfusion
+description: Checkout and learn here all about Detail Template in Syncfusion Blazor Tree Grid component and much more details.
+platform: Blazor
+control: Tree Grid
+documentation: ug
+---
+
+# Detail Template in Blazor Tree Grid Component
+
+The detail template provides additional information about a particular row. By expanding the parent row the child rows are expanded along with their detail template. The [DetailTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor~Syncfusion.Blazor.TreeGrid.SfTreeGrid~DetailTemplate.html) property accepts either the template string or HTML elements.

```

Comment:
The detail template provides additional information about a particular row. By expanding the parent row, the child rows are expanded along with their detail template. The [DetailTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor~Syncfusion.Blazor.TreeGrid.SfTreeGrid~DetailTemplate.html) property accepts either the template string or HTML elements.